### DISCUSSION TITLE
Possibility to disable "Edit Raw Value"

### DISCUSSION THREAD

**Discussion by Michael-Nussbaumer** [directus/directus#16378] [Feature Requests]
In my opinion this should be disableable, as this can create huge problems if inexperienced users start to use that feature.

**Comment by preetamslot** [comment]
@MichaelNussbaumerGOWEST 

You can disable it with CSS for all users.

```
#main-content .field .field-label {
   pointer-events: none; 
}
```

**Reply by Dominic-Preap** [reply]
You got me the easiest way. Thank you

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
